test/e2e/basic_test.go

```diff
@@ -58,6 +59,7 @@ func TestBasic(t *testing.T) {
 	t.Run("testGratuitousARP", func(t *testing.T) { testGratuitousARP(t, data, data.testNamespace) })
 	t.Run("testClusterIdentity", func(t *testing.T) { testClusterIdentity(t, data) })
 	t.Run("testLogRotate", func(t *testing.T) { testLogRotate(t, data) })
+	t.Run("testForwardCompatibility", func(t *testing.T) { testForwardCompatibility(t, data) })
```

Comment:
nit: I would just suggest renaming the test to `testConfigForwardCompatibility`

Reply:
renamed, thanks